src/Core/SequenceExtensions.fs

```diff
@@ -61,3 +61,8 @@ let lastButOne seq =
     |> Seq.rev
     |> Seq.skip 1
     |> Seq.head
+
+let isMultiple seq =
```

Comment:
I would say `containsMoreThanOne` or `containsMultiple`

Reply:
That was an attempt to follow the `isEmpty` name, but I agree, `containsMultiple` is a better one anyway - updated.